recipes-sota/aktualizr/files/aktualizr-secondary.socket

```diff
@@ -1,5 +1,6 @@
 [Socket]
 ListenStream=9030
+ListenDatagram=9030
```

Comment:
Not in sync with the current aktualizr default of 9031 for discovery. We should probably make them match to avoid confusion.